index.html

```diff
@@ -788,6 +849,99 @@ <h4>`ThingLink`</h4>
                 </section>
             </section>
         </section>
+
+        <section id="exploration-service" class="normative">
+            <h2>Service</h2>
+            <p>
+             Any web service that can be referenced by a URL and return a TD with
+             appropriate authentication and access controls can be used as an
+             exploration mechanism.  
+             This service does not need to be a Thing.
+             In particular, a TD can be hosted on an ordinary web server
+             and referenced by its URL.  
+            </p>
+            <p>
+             However, a service distributing a TD 
+             using the following protocols 
+             is subject to the following constraints:
+            </p> 
+            <dl>
+                <dt>HTTP</dt>
+                <dd>
+                        <p class="ednote" title="Redundant">
+                        The following two assertions on TLS and authentication
+                        are redundant and not as specfic as those provided elsewhere.
+                        Suggest replacing these here (and in all exploration sections) with appropriate
+                        references to security and privacy assertions given 
+                        in [[wot-architecture11]] and [[wot-thing-description11]]
+                        and in the Security Considerations and Privacy Considerations sections of this document.
+                        </p>
+                    <p>
+                        <span class="rfc2119-assertion" id="service-http-secure"> 
+                            An HTTP server providing a <a>TD</a> 
+                            SHOULD be using TLS.
```

Comment:
TLS for?
```suggestion
                            An HTTP server providing a <a>TD</a> SHOULD use TLS when serving requests.
```

Reply:
I will accept, although this whole assertion will be going away; I want to replace it with a reference to assertions we already have under Security and Privacy considerations.